src/runtime/io.cpp

```diff
@@ -289,22 +289,6 @@ extern "C" obj_res lean_io_prim_handle_is_eof(b_obj_arg h, obj_arg /* w */) {
     return io_result_mk_ok(box(std::feof(fp) != 0));
 }
 
-/* Handle.size : (@& Handle) → IO USize */
```

Comment:
This function was incorrect since we cannot compute the size of `Handle` in general ([previous discussion](https://github.com/leanprover/lean4/pull/235#discussion_r548086989)).